EIPS/eip-execute-past-semantic-abort.md

```diff
@@ -0,0 +1,116 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 4511
```

Reply:
Make sure to rename the file to `EIPS/eip-4511.md` as well.